tests/test_nomicbert.py

```diff
@@ -0,0 +1,163 @@
+
+As the 1990s approached, the introduction of 16-bit consoles, exemplified by the Sega Genesis and the Super Nintendo, marked a significant leap in graphical fidelity and sound design. Games could now convey more complex stories and offer richer visual experiences, leading to a golden age of creativity and innovation. This period saw the birth of genre-defining titles like "Sonic the Hedgehog," which combined breathtaking speed with memorable level design, and "Final Fantasy VI," a game that showcased the potential for video games to deliver epic narratives and emotional depth. The battle between Sega and Nintendo during this era spurred rapid advancements in technology and game design, a competitive spirit that benefited gamers with a plethora of high-quality titles to choose from. The resurgence of arcades was also notable, with fighting games like "Street Fighter II" and "Mortal Kombat" drawing crowds with their intense, competitive gameplay and colorful, often controversial characters. This decade also saw the advent of digital pets and life simulation games like "Tamagotchi".
+"""
+
+
+if __name__ == "__main__":
+    MODEL_NAME = "nomic-ai/nomic-embed-text-v1.5"
+    tokenizer = AutoTokenizer.from_pretrained("bert-base-uncased")
+    long_inputs = tokenizer(long_input, return_tensors="pt", padding=True)
+    num_tokens = long_inputs['input_ids'].shape
+
+    print(f"Number of tokens in the input: {num_tokens}")
+
+    hf_model = AutoModel.from_pretrained(
+        MODEL_NAME,
+        trust_remote_code=True,
+        safe_serialization=True,
+        revision="91d2d6bfdddf0b0da840f901b533e99bae30d757",
+        force_download=True
+    )
+
+    onnx_path = Path("tests/nomic_bert.onnx")
+    onnx_config_constructor = TasksManager.get_exporter_config_constructor(
+        "onnx", hf_model, task="feature-extraction", library_name="transformers"
+    )
+    onnx_config = onnx_config_constructor(hf_model.config)
+    onnx_inputs, onnx_outputs = export(hf_model, onnx_config, onnx_path, onnx_config.DEFAULT_ONNX_OPSET)
+
+    print("After export")
+    print(onnx_inputs)
+    print(onnx_outputs)
```

Comment:
`nomic_bert.onnx` is not defined.

The way to go is to use `tempfile.TemporaryDirectory()` and https://huggingface.co/docs/optimum/main/en/exporters/onnx/package_reference/export#optimum.exporters.onnx.main_export